src/core/meshcop/tcat_agent.hpp

```diff
@@ -374,6 +390,7 @@ class TcatAgent : public InstanceLocator, private NonCopyable
     static constexpr uint16_t kJoinerUdpPort            = OPENTHREAD_CONFIG_JOINER_UDP_PORT;
     static constexpr uint16_t kPingPayloadMaxLength     = 512;
     static constexpr uint16_t kProvisioningUrlMaxLength = 64;
+    static constexpr uint16_t kMaxPSKdLength            = OT_JOINER_MAX_PSKD_LENGTH;
```

Comment:
Per OT naming style guide, we treat abbreviation as nouns and use`CamelCase` style for all, e.g. `UdpSocket`, `SrpClient`, Pskc`, ect.
```suggestion
    static constexpr uint16_t kMaxPskdLength            = OT_JOINER_MAX_PSKD_LENGTH;
```

Reply:
fixed.